lib/active_model/serializer.rb

```diff
@@ -73,7 +73,7 @@ def self.serializer_lookup_chain_for(klass, namespace = nil)
     # Used to cache serializer name => serializer class
     # when looked up by Serializer.get_serializer_for.
     def self.serializers_cache
-      @serializers_cache ||= ThreadSafe::Cache.new
+      @serializers_cache ||= Concurrent::Map.new
```

Comment:
seems out of scope

Reply:
for reference: `thread_safe` gem is now deprecated and merged into `concurrent-ruby`

commit: https://github.com/ruby-concurrency/concurrent-ruby/commit/52e5f373a93abbe1f0339cf3087a269e4a76ff3e#diff-42d5a45da331eaa07d2b315bd3c9e738

Reply:
i will move this change to another PR if you like